articles/azure/azs-how-create-scale-set.md

```diff
@@ -0,0 +1,126 @@
+---
+title: How to Create a Scale Set using UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides help for creating a scale set on UKCloud for Microsoft Azure
+services: azure-stack
+author: Shaion O'Niel
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a Scale Set - portal
+toc_fullpath: Users/How To/azs-how-create-scale-set.md
+toc_mdlink: azs-how-create-scale-set.md
+---
+
+# How to create a Virtual Machine Scale Set using UKCloud Azure Stack portal
+
+## Overview
+
+Virtual Machine Scale Sets in Azure Stack are a compute resource that can be used to deploy and manage identical virtual machines. All virtual machine instances in a Scale Set are configured in the same way and do not require any provisioning beforehand.
+
+## Prerequisites
+
+To complete the steps in this guide, you must have appropriate access to a subscription in the Azure Stack portal.
+
+## Creating a new virtual machine scale set
+
+1. Log in to the Azure Stack portal.
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. In the favourites panel, select **Create a resource**.
+
+    ![New option in favourites panel](images/azsp_newmenu.png)
+
+3. In the *Everything* blade search **'Virtual machine scale set'**.
+
+    ![Searching for scale set](images/azs-browser-button-scale-set-menu.png)
+
+4. In the *virtual machine scale set* blade select **Create**.
+
+    ![Create scale set button](images/azs-browser-button-create-scale-set.png)
+
+5. In the *BASICS* page, enter the following information:
+
+    - **Virtual machine scale set name** - The name of the Scale Set.
+
+    - **Operating system disk image** – The operating system disk image for the virtual machines in the scale set.
+
+    > [!Warning]
+    > Scale Sets currently do not support CentOS 7.2 on Azure Stack.
+
+    - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+    - **Location** - This will be frn00006, which is the location of the Azure Stack.
+
+    - **Username** – Admin username for the virtual machines.
+
+    - **Password** – Admin password for the virtual machines.
+
+    - **Confirm password** – Confirm the password.
+
+    - **Instance count** – The number of virtual machines in the scale set (0 - 100).
+
+    - **Instance** size – The size of each virtual machine in the scale set. Some sizes may only be available using templates, PowerShell, or CLI due to the recommended portal defaults.
+
+    - **Use managed disks** – Managed disks hide the underlying storage accounts and instead shows the abstraction of a disk. Unmanaged disks expose the underlying storage accounts and VHD blobs.
```

Comment:
Can we change it so something like:
Managed Disks is a feature that simplifies disk management for Azure IaaS VMs by handling storage account management for you. For more information, see the Managed Disks overview.
Managed Disks overview -> https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview
Do a proper link of course...